beetsplug/fetchart.py

```diff
@@ -447,6 +502,13 @@ def __init__(self):
         if not self.config['google_key'].get() and \
                 u'google' in available_sources:
             available_sources.remove(u'google')
+        if not self.config['fanarttv_key'].get() and \
+                u'fanarttv' in available_sources:
```

Comment:
That's a good point—we're not usually in the habit of duplicating the docs with a run-time warning unless something's serious. Let's turn this off.